rules/docker-platform-monitor.mk

```diff
@@ -61,5 +61,5 @@ $(DOCKER_PLATFORM_MONITOR)_BASE_IMAGE_FILES += cmd_wrapper:/usr/sbin/SmartCmd
 $(DOCKER_PLATFORM_MONITOR)_BASE_IMAGE_FILES += cmd_wrapper:/usr/bin/ethtool
 $(DOCKER_PLATFORM_MONITOR)_FILES += $(SUPERVISOR_PROC_EXIT_LISTENER_SCRIPT)
 
-SONIC_BUSTER_DOCKERS += $(DOCKER_PLATFORM_MONITOR)
-SONIC_BUSTER_DBG_DOCKERS += $(DOCKER_PLATFORM_MONITOR_DBG)
+SONIC_BULLSEYE_DOCKERS += $(DOCKER_PLATFORM_MONITOR)
```

Comment:
By moving pmon to bullseye isn't it required for all vendors to support bullseye? Unlike syncd pmon is common right?

Reply:
Not exactly... all vendors platform API must support python 3 is the only restriction that I am aware of.